### PR TITLE
Touch For Mobile

### PR DESCRIPTION
**Pull request by SarahPappas**
- replace mouse events with pointer events
- increase radius of snapping from 10 to 20
- fix name bug

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>BobbinLab</title>
+    <title>Bobbin Lab</title>
```
Comment (msdrey) [inline public/index.html]: Did you want to go back from BobbinLab to Bobbin Lab?

File: public/manifest.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "BobbinLab",
-  "name": "BobbinLab",
+  "short_name": "Bobbin Lab",
+  "name": "Bobbin Lab",
```
Comment (msdrey) [inline public/manifest.json]: see previous comment

File: public/index.html (hunk 1 above)
Comment (SarahPappas) [reply public/index.html]: I changed these back! Sorry

File: public/manifest.json (hunk 2 above)
Comment (SarahPappas) [reply public/manifest.json]: I changed this back.

**Pull request merged**